packages/python-pulp-cli-deb/python-pulp-cli-deb.spec

```diff
@@ -0,0 +1,68 @@
+%{?scl:%scl_package python-%{pypi_name}}
+%{!?scl:%global pkg_name %{name}}
```

Comment:
What is this for? (Does not exist in the python-pulp-cli package.)

Reply:
Not sure, I just ran this script: https://github.com/theforeman/foreman-packaging/blob/461c8df52c17de853a8a0c0657a7d18135a73229/add_pypi_package.sh

Reply:
@evgeni?

Reply:
to build the package inside the SCL, so we get Python 3.8, not 3.6